package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "To do list is a basic website, On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --watch --mode development",
+    "dev": "webpack-dev-server --mode=development"
```

Comment:
- [ ] I cannot open the webpack server because 'start' script is missing. When users run 'npm run start' a development server  should open on browser. Please make sure to go through all all [webpack installation steps](https://webpack.js.org/guides/getting-started/)

Reply:
Fixed